pkg/flow/reconciler/testing/listers.go

```diff
@@ -87,11 +92,16 @@ func (l *Listers) GetServingObjects() []runtime.Object {
 	return l.sorter.ObjectsForSchemeFunc(fakeservingclient.AddToScheme)
 }
 
-// GetXSLTTransformationLister returns a Lister for GoogleSheetTarget objects.
+// GetXSLTTransformationLister returns a Lister for XSLTTransformation objects.
 func (l *Listers) GetXSLTTransformationLister() flowlisters.XSLTTransformationLister {
 	return flowlisters.NewXSLTTransformationLister(l.IndexerFor(&flowv1alpha1.XSLTTransformation{}))
 }
 
+// GetJQTransformationLister returns a Lister for JQTransformation objects.
+func (l *Listers) GetJQTransformationLister() flowlisters.JQTransformationLister {
```

Comment:
(related to previous comment)

Not needed. We could just collect all flow objects here:
https://github.com/triggermesh/triggermesh/blob/0ffd100f029a91d7367a8e5da5192909e3c87402/pkg/sources/reconciler/testing/listers.go#L87-L89

Reply:
This one I have a little more of an objection to as my understanding is that the `Lister` is specific to the type as per: https://github.com/triggermesh/triggermesh/blob/0fa76cf2bd91f11c9d211f3098fe584dcafaf3b6/pkg/sources/reconciler/testing/listers.go#L107 (referencing one of the many listeres defined for the reconciler lister).

Reply:
The two are unrelated. The lister you're referring to is used in a different context.

Reply:
Is it?  I must be missing something because I'm referencing it in a similar context. Unless you're referring to mine using the `NewKServiceReconciler` function to initialize the reconciler.

Reply:
Sorry I got completely mixed up with my previous comment on the Factory and the usage of `GetFlowObjects()` 🤦
Yes you're right, we also need this one of course.

Reply:
Whew, glad it wasn't me for a change. :)